docker-compose/docker-compose.yml

```diff
@@ -89,12 +89,23 @@ services:
     volumes:
       - rabbitmq-volume:/var/lib/rabbitmq # default data dir
 
+  seq:
```

Comment:
nothing depends on this

Reply:
I think that's okay. If the server is not available yet, I think our logging library waits until it is. And if it doesn't it's okay as well, as the startup logs are not that important.
Waiting would increase the startup time though, which would be worse.

Reply:
Though... Now that I think about it: it would be nice if the seq service would be started automatically when you start the consumer api 🤔

Reply:
But as I don't have time now to wait for the pipelines to run again, I will do it as part of my next PR.